README.md

```diff
@@ -373,57 +403,71 @@ Set this property to `false` to disable this plugin.
 
 _No options are available for this plugin._
 
-### youtube
+### caniuse
 
-Add support for [YouTube](http://youtube.com/) embeds in Markdown, as block syntax.
+Add support for [CanIUse](https://caniuse.com/) embeds in Markdown, as block syntax.
+Uses https://caniuse.bitsofco.de/ to provide interactive embeds from CanIUse data.
 
-The basic syntax is `[youtube <id>]`. E.g. `[youtube iom_nhYQIYk]`.
-Height and width can optionally be set using `[youtube <id> [height] [width]]`. E.g. `[youtube iom_nhYQIYk 380 560]`.
-The default value for height is 270, and for width is 480.
+The basic syntax is `[caniuse <feature>]`. E.g. `[caniuse css-grid]`.
+After the slug, some space-separated flags can be added (in any combination/order):
+
+- Add `past=` followed by a number to control how many previous browser versions to include (default is 1, supported 0-5).
+- Add `future=` followed by a number to control how many previous browser versions to include (default is 1, supported 0-3).
+- Add `accessible` to set the default color scheme for the CanIUse embed to be accessible colors.
 
 **Example Markdown input:**
 
-    [youtube iom_nhYQIYk]
+    [caniuse css-grid]
+
+    [caniuse css-grid past=5 future=3 accessible]
 
 **Example HTML output:**
 
-    <iframe src="https://www.youtube.com/embed/iom_nhYQIYk" class="youtube" height="380" width="560" frameborder="0" allowfullscreen>
-        <a href="https://www.youtube.com/watch?v=iom_nhYQIYk" target="_blank">View YouTube video</a>
-    </iframe>
+    <p class="ciu_embed" data-feature="css-grid" data-periods="future_1,current,past_1" data-accessible-colours="false">
```

Comment:
Should we default to the American English spelling of colors?

Reply:
Alas we have no control over that, 'tis what the embed script we're importing expects: https://github.com/ireade/caniuse-embed/blob/9eeec520c0096120e6c131e258ba4d3fa1d5bbfc/src/caniuse-embed.js#L11

Reply:
Ah, very good. Disregard my numerous comments on that then.